olive/systems/local.py

```diff
@@ -19,6 +19,9 @@
 class LocalSystem(OliveSystem):
     system_type = SystemType.Local
 
+    def __init__(self, accelerators: List[AcceleratorConfig] = None, hf_token: bool = None):
```

Comment:
Is there a reason this is added? Since it's just calling the super()'s init, this is redundant.

Reply:
Good catch. Forgot to remove this, this was added for remote dispatcher.